### PR TITLE
feat(about-us): add features to add, and edit about-us  data

### PR DESCRIPTION
**Pull request by febrian-andi**

### CODE REVIEW

**Comment by ignasiusadhitia** [comment]
> **The latest updates on your projects**. Learn more about [Vercel for Git ↗︎](https://vercel.link/github-learn-more)
> 
> Name	Status	Preview	Comments	Updated (UTC)
> **kelompok-1c**	✅ Ready ([Inspect](https://vercel.com/ignasiusadhitias-projects/kelompok-1c/FgThMShYM2WusN1PgHFTrst3jZfY))	[Visit Preview](https://vercel.live/open-feedback/kelompok-1c-git-feature-about-us-ignasiusadhitias-projects.vercel.app?via=pr-comment-visit-preview-link&passThrough=1)	💬 [**Add feedback**](https://vercel.live/open-feedback/kelompok-1c-git-feature-about-us-ignasiusadhitias-projects.vercel.app?via=pr-comment-feedback-link)	Nov 16, 2024 6:31pm

silakan tes dulu mas @febrian-andi 

https://kelompok-1c-git-feature-about-us-ignasiusadhitias-projects.vercel.app/?_vercel_share=1956qEpWJs0DLDcG37UKlwgEwrXWCyFH

File: src/App.jsx [hunk 1]
```diff
@@ -15,7 +15,8 @@ import TeamPage from 'pages/TeamPage';
 import TestimonialPage from 'pages/TestimonialPage';
 import WhatWeDoPage from 'pages/WhatWeDoPage';
 
-import AboutUsFormPage from './pages/AboutUsFormPage';
+import AddAboutUsFormPage from './pages/AboutUs/AddAboutUsFormPage';
```
Comment (ignasiusadhitia) [inline src/App.jsx]:
```suggestion
import AddAboutUsFormPage from 'pages/AboutUs/AddAboutUsFormPage';
```

File: src/App.jsx [hunk 2]
```diff
@@ -15,7 +15,8 @@ import TeamPage from 'pages/TeamPage';
...
+import EditAboutUsFormPage from './pages/AboutUs/EditPortfolioFormPage';
```
Comment (ignasiusadhitia) [inline src/App.jsx]:
```suggestion
import EditAboutUsFormPage from 'pages/AboutUs/EditPortfolioFormPage';
```

File: src/components/AboutUs/AboutUsForm.jsx [hunk 3]
```diff
@@ -1,7 +1,146 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import useAddData from '../../hooks/useAddData';
+import useUpdateData from '../../hooks/useUpdateData';
```
Comment (ignasiusadhitia) [inline src/components/AboutUs/AboutUsForm.jsx]:
```suggestion
import useAddData from 'hooks/useAddData';
import useUpdateData from 'hooks/useUpdateData';
```

File: src/containers/AboutUs/AboutUsContainer.jsx [hunk 4]
```diff
@@ -1,7 +1,78 @@
 import React from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import useFetchData from '../../hooks/useFetchData';
+import AboutUsList from '../../components/AboutUs/AboutUsList';
```
Comment (ignasiusadhitia) [inline src/containers/AboutUs/AboutUsContainer.jsx]:
```suggestion
import useFetchData from 'hooks/useFetchData';
import AboutUsList from 'components/AboutUs/AboutUsList';
```

File: src/pages/AboutUs/AboutUsPage.jsx [hunk 5]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import AboutUsContainer from '../../containers/AboutUs/AboutUsContainer';
```
Comment (ignasiusadhitia) [inline src/pages/AboutUs/AboutUsPage.jsx]:
```suggestion
import AboutUsContainer from 'containers/AboutUs/AboutUsContainer';
```

**Comment by febrian-andi** [comment]
Preview nya gak bisa dibuka mas @ignasiusadhitia . Butuh request access.
Screenshot 2024-11-17 023024

**Comment by ignasiusadhitia** [comment]
> Preview nya gak bisa dibuka mas @ignasiusadhitia . Butuh request access. Screenshot 2024-11-17 023024

sudah ya, silakan coba lagi

**Comment by febrian-andi** [comment]
@ignasiusadhitia Saya coba aman mas
image

**Comment by ignasiusadhitia** [comment]
> @ignasiusadhitia Saya coba aman mas image

image
image
image


ini saya beberapa kali coba kena 400 terus, invalid token. padahal sudah beberapa kali buat akun baru untuk tes about section

**Review by ignasiusadhitia — APPROVED** [review]
okay aman. well done

image

**Pull request merged**